src/System.Private.Xml/src/System/Xml/XPath/Internal/XPathArrayIterator.cs

```diff
@@ -84,7 +84,5 @@ public override IEnumerator GetEnumerator()
         {
             return list.GetEnumerator();
         }
-
-        private object debuggerDisplayProxy { get { return index < 1 ? null : (object)new XPathNavigator.DebuggerDisplayProxy(Current); } }
```

Comment:
As I previously commented, this must not be deleted.  It's being used by the debugger display attribute... see the attribute on the type declaration.